reddit_decider/__init__.py

```diff
@@ -826,40 +795,62 @@ def get_all_dynamic_configs(self) -> List[Dict[str, Any]]:
 
         :return: list of all active dynamic config dicts.
         """
-        decider = self._get_decider()
-        if not decider:
-            return []
-
-        ctx = self._get_ctx()
-        ctx_err = ctx.err()
-        if ctx_err is not None:
-            logger.info(f"Encountered error in rust_decider.make_ctx(): {ctx_err}")
+        if self._internal is None:
+            logger.error("rs_decider is None--did not initialize.")
             return []
 
-        all_decisions_result = decider.get_all_values(ctx)
+        ctx = self._decider_context.to_dict()
 
-        error = all_decisions_result.err()
-        if error:
-            logger.info(f"Encountered error in decider.choose_all(): {error}")
+        try:
+            values = self._internal.all_values(ctx)
+        except DeciderException as exc:
+            logger.info(str(exc))
             return []
 
-        all_decisions = all_decisions_result.decisions()
         parsed_configs = []
 
-        for dc_name, decision in all_decisions.items():
-            decision_error = decision.err()
-            if decision_error:
-                logger.info(
-                    f"Encountered error for dynamic config: {dc_name} in decider.get_all_values(): {decision_error}"
-                )
-                continue
+        for feature_name, val in values.items():
+            parsed_configs.append(self._value_to_dc_dict(feature_name, val))
 
-            value_dict = decision.value_dict()
+        return parsed_configs
+
+    def _get_dynamic_config_value(
+        self,
+        feature_name: str,
+        default: Any,
+        dc_type: Type[T],
+    ) -> T:
+        if self._internal is None:
+            logger.error("rs_decider is None--did not initialize.")
+            return default
 
-            if value_dict:
-                parsed_configs.append(value_dict)
+        ctx = self._decider_context.to_dict()
 
-        return parsed_configs
+        try:
+            value = eval(f"self._internal.get_{dc_type.__name__}")(
+                feature_name=feature_name, context=ctx
+            )
```

Comment:
Same idea as the other PR. If we're going to pass a thing in here, might as well pass the actual method instead of adding the type and then evaling.